picard/disc.py

```diff
@@ -19,6 +20,26 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
+import os
+import sys
+import traceback
+
+from PyQt5 import QtCore
+
+from picard import log
+from picard.const.sys import (
+    IS_FROZEN,
+    IS_MACOS,
+)
+
+from picard.ui.cdlookup import CDLookupDialog
+
+
+# Ensure libdiscid.dylib gets loaded from app bundle
+if IS_MACOS and IS_FROZEN:
```

Comment:
Should this be a runtime hook?

Reply:
I guess we could move it to a runtime hook, yes.

Reply:
It would be great if we could do that. Move as much freezing logic to hooks. That way we can keep the module logic relatively unpolluted with pyinstaller logic.

Reply:
Here we go: https://github.com/metabrainz/picard/pull/1348